grails-app/taglib/org/pih/warehouse/RenderMessageTagLib.groovy

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2012 Partners In Health.  All rights reserved.
+ * The use and distribution terms for this software are covered by the
+ * Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+ * which can be found in the file epl-v10.html at the root of this distribution.
+ * By using this software in any fashion, you are agreeing to be bound by
+ * the terms of this license.
+ * You must not remove this notice, or any other, from this software.
+ **/
+package org.pih.warehouse
+
+/**
+ * available attributes:
+ * * style - style of a message container ex: message, error (default: message)
+ * * message - a single or a list of messages to be rendered in the template
+ * by default takes the value stored in flash.message.
+ * accepts data like: String, List<String>, Map<code: string, args: obj>, List<Map<code: string, args: obj>>
+ */
+class RenderMessageTagLib {
```

Comment:
@drodzewicz I am not sure if this is a good place for this tag lib (if there is need for a new tag lib). I wonder if this rather should be in the `MessageTagLib`.